From andre@bitwigglers.org  Thu Jun 19 12:06:43 2008
From: Andre Haupt <andre@bitwigglers.org>
Date: Wed, 18 Jun 2008 15:56:00 +0200
Subject: USB: usb-serial: fix a sparse warning about different signedness
To: linux-usb@vger.kernel.org
Cc: greg@kroah.com, Andre Haupt <andre@bitwigglers.org>
Message-ID: <502465cf27898887ddf30859df9420ab8e2c48b1.1213796327.git.andre@bitwigglers.org>


fix the following sparse warning:

drivers/usb/serial/usb-serial.c:927:43: warning: incorrect type in argument 3 (different signedness)
drivers/usb/serial/usb-serial.c:927:43:    expected unsigned int *minor
drivers/usb/serial/usb-serial.c:927:43:    got int *<noident>
  CHECK   drivers/usb/serial/generic.c

Signed-off-by: Andre Haupt <andre@bitwigglers.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/serial/usb-serial.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/usb/serial/usb-serial.c
+++ b/drivers/usb/serial/usb-serial.c
@@ -625,7 +625,7 @@ int usb_serial_probe(struct usb_interfac
 	struct usb_endpoint_descriptor *bulk_out_endpoint[MAX_NUM_PORTS];
 	struct usb_serial_driver *type = NULL;
 	int retval;
-	int minor;
+	unsigned int minor;
 	int buffer_size;
 	int i;
 	int num_interrupt_in = 0;
